Advanced Communication Networks/adv_comnet.tex

```diff
@@ -896,7 +898,7 @@
 \end{lstlisting}
 
   \subsubsection{Tuple}
-  Unordered collection of unnamed members.
+  Ordered collection of unnamed members.
```

Comment:
There is Unordered written in the slides, but as far as I understood, tuples must always be ordered.